### PR TITLE
README.md and requirements.txt added

### PR DESCRIPTION
**Pull request by abenezer-gelaw**

### CODE REVIEW

File: consistency/tool/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+}
+```
+The ConsistencyReport formal definition and meaning of associated terms can be found [here](https://github.com/apigee/registry-experimental/blob/main/google/cloud/apigeeregistry/applications/v1alpha1/consistency/consistency_report.proto).
+
+Given a spec name, this part of the tool creates a report about the word usage in the spec compared to the entire project. In this step, we detect word groups similar to each word found in the spec. We also detect unique words just introduced by a spec into the project.
+
+### Interface: 
+Once a comparison report has been generated for one (or several) specs, we generate an easily understood output csv file for users. In this step, the results we computed in the step above are finally provided in human-readable file(s). 
+
+## Getting Started
```
Comment (shrutiparabgoogle) [inline consistency/tool/README.md:50]: In the getting started step, can we include `make py-protos` as well? That we are done with the setup.

Example usage should then include only steps which demonstrate how to use the tool.

File: consistency/tool/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,38 @@
+black==22.6.0
```
Comment (shrutiparabgoogle) [inline consistency/tool/requirements.txt:1]: Curious to know how this file was generated

**Review by Krpate — APPROVED** [review]

File: consistency/tool/README.md (hunk 1 above)
Comment (shrutiparabgoogle) [reply consistency/tool/README.md:50]: Can you also put a link for the use of pyenv https://cloud.google.com/python/docs/setup#installing_python

File: .gitignore [hunk 3]
```diff
@@ -7,3 +7,6 @@ consistency/gnostic-protos
 third_party
 .vscode/
 my_env/
+consistency/tool/interface/__pycache__
```
[the remark this answers is not in the document]
Comment (abenezer-gelaw) [reply .gitignore:10]: Running the tool creates this cache folder by default for optimization. I was hoping to exclude it...

**Pull request merged**